packages/beacon-node/src/network/reqresp/handlers/index.ts

```diff
@@ -20,13 +28,25 @@ export function getReqRespHandlers({db, chain}: {db: IBeaconDb; chain: IBeaconCh
     async *onStatus() {
       yield chain.getStatus();
     },
-
     async *onBeaconBlocksByRange(req) {
       yield* onBeaconBlocksByRange(req, chain, db);
     },
-
     async *onBeaconBlocksByRoot(req) {
       yield* onBeaconBlocksByRoot(req, chain, db);
     },
+    async *onLightClientBootstrap(req) {
+      yield* onLightclientBootstrap(req, chain);
+    },
+    async *onLightClientUpdate(req) {
+      // TODO DA confirm MAX_REQUEST_LIGHT_CLIENT_UPDATES is adhered to.
+      // TODO DA Harmonize the capitalization of lightClient vs lightclient
+      yield* onLightclientUpdate(req, chain);
+    },
+    async *onLightClientFinalityUpdate() {
+      yield* onLightClientFinalityUpdate(chain);
+    },
+    async *onLightClientOptimisitcUpdate() {
```

Comment:
i am not sure a peer would do req/resp over the header as the new header should be easily available over `gossip`, but we can still keep it as the marginal cost to support this is `nil`